src/modules/involvementAPI/addLikes.js

```diff
@@ -0,0 +1,16 @@
+import { getLikes } from './getLikes.js';
+
+const addLikes = async () => {
+  const likes = await getLikes();
+  const likesContainer = document.querySelectorAll('.likes');
+  likesContainer.forEach((like) => {
+    const id = like.parentElement.parentElement.parentElement.parentElement
+      .firstElementChild.textContent;
+    const movieLikes = likes.filter(
+      (item) => parseInt(item.item_id, 10) === parseInt(id, 10),
+    );
+    like.textContent = movieLikes.length === 0 ? 0 : movieLikes[0].likes;
+  });
+};
+
```

Comment:
- [ ] I think it would be better that if a user clicks on the `like` button, the  `likes counter` increments instantly without having to refresh the page to see the change. This can be achieved by fetching data from the `involvement API` whenever the `like` button is clicked. This can help improve the user experience.  Also, you could cover the `likes counter`  in your tests as it is required in this milestone that all counters should be tested

Reply:
Thank you Frank for the review. 
We have fixed the page to show likes increment on clicking. However, testing for the likes counter was not included in the project cards for the required activities.